src/email/templates/orderConfirmation.ejs

```diff
@@ -0,0 +1,132 @@
+                height: auto !important;
+                max-width: 100% !important;
+                width: auto !important;
+            }
+        }
+
+        /* -------------------------------------
+            PRESERVE THESE STYLES IN THE HEAD
+        ------------------------------------- */
+        @media all {
+            .ExternalClass {
+                width: 100%;
+            }
+            .ExternalClass,
+            .ExternalClass p,
+            .ExternalClass span,
+            .ExternalClass font,
+            .ExternalClass td,
+            .ExternalClass div {
+                line-height: 100%;
+            }
+            .apple-link a {
+                color: inherit !important;
+                font-family: inherit !important;
+                font-size: inherit !important;
+                font-weight: inherit !important;
+                line-height: inherit !important;
+                text-decoration: none !important;
+            }
+            .btn-primary table td:hover {
+                background-color: #34495e !important;
+            }
+            .btn-primary a:hover {
+                background-color: #34495e !important;
+                border-color: #34495e !important;
+            }
+        }
+    </style>
+</head>
+<body class="" style="background-color: #f6f6f6; font-family: sans-serif; -webkit-font-smoothing: antialiased; font-size: 14px; line-height: 1.4; margin: 0; padding: 0; -ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%;">
+<table border="0" cellpadding="0" cellspacing="0" class="body" style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%; background-color: #f6f6f6;">
+    <tr>
+        <td style="font-family: sans-serif; font-size: 14px; vertical-align: top;">&nbsp;</td>
+        <td class="container" style="font-family: sans-serif; font-size: 14px; vertical-align: top; display: block; Margin: 0 auto; max-width: 580px; padding: 10px; width: 580px;">
+            <div class="content" style="box-sizing: border-box; display: block; Margin: 0 auto; max-width: 580px; padding: 10px;">
+
+<% for(var i=0; i < order.items.length; i++) { %>
+                    <p><%= order.items[i].itemName %></p>
+                    <p><small><%= order.items[i].salePrice %> x <%= order.items[i].quantityRequested %> = <%= order.items[i].total %></small></p>
+<% } %>
+    <h1><%= order.items[i].totalCost %></h1>
+                                        <p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">To pick up your items, find a merch distribution event on the membership portal and stop by.</p>
+                                        <p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">Regards, <br />ACM UCSD</p>
+                                    </td>
+                                </tr>
+                            </table>
```

Comment:
Getting an error caused by these lines when ordering any item. The order still goes through but I don't get any confirmation email.
```
InternalServerError: ejs:100
    98|                     <p><small><%= order.items[i].salePrice %> x <%= order.items[i].quantityRequested %> = <%= order.items[i].total %></small></p>
    99| <% } %>
 >> 100|     <h1><%= order.items[i].totalCost %></h1>
    101|                                         <p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">To pick up your items, find a merch distribution event on the membership portal and stop by.</p>
    102|                                         <p style="font-family: sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 15px;">Regards, <br />ACM UCSD</p>
    103|                                     </td>

Cannot read property 'totalCost' of undefined
```

Input request:
```
{
	"order": [
		{
			"item": "6b65bec1-4bff-4334-bf64-792b83ed426a",
			"quantity": 1
		}
	]
}
```
`order` response when console.logged:
```
{
  uuid: '1ea1fbee-7aa4-4257-b869-d3a7173e51ce',
  orderedAt: 2020-05-20T18:56:15.360Z,
  totalCost: 1275,
  items: [
    {
      itemName: 'Hack School Sticker Pack (4)',
      picture: null,
      description: 'Make space on your laptop cover for these 2 in x 2 in stickers. Pack of 4.',
      quantityRequested: 1,
      salePrice: 1275,
      total: 1275
    }
  ]
```

Reply:
Fixed typo.